composer.json

```diff
@@ -12,7 +12,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": ">=7.4",
+        "php": ">=7.4 || ^8.0",
```

Comment:
No need to change it here

Reply:
Fixed, thanks for your review

Reply:
Actually >=7.4 was fine, why changing it to this?